sdk/modelsrepository/azure-iot-modelsrepository/src/test/java/com/azure/iot/modelsrepository/FetchPreminilaryTests.java

```diff
@@ -0,0 +1,16 @@
+package com.azure.iot.modelsrepository;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+public class FetchPreminilaryTests {
```

Comment:
Fetch**Preliminary**Tests

Reply:
file removed all together :D I can't spell for ***